### DISCUSSION TITLE
VsCode Workspace file Variable reference

### DISCUSSION THREAD

**Discussion by Satheeshkumar-Duraisamy** [gitkraken/vscode-gitlens#1426] [Q&A]
What is the variable reference to get VsCode Workspace file **xyz.code-workspace** directory?

**Comment by eamodio** [comment]
Sorry, I don't understand the question.

**Reply by Satheeshkumar-Duraisamy** [reply]
The VSCode internal variable which gives path to xyz.code-workspace file
